rooutil/inc/common_cuts.hh

```diff
@@ -191,8 +191,8 @@ bool three_of_four_coinc(CrvCoinc& crv_coinc) { // CRV coincidence has exactly t
 }
 
 //+ Combined Track & CrvCoinc Cuts
-bool track_crv_coincidence(TrackSegment& segment, CrvCoinc& crv_coinc) { // time difference between track segment and crv_coinc is less than 250 ns
-  if ( segment.trkseg->time - crv_coinc.reco->time > 200 && segment.trkseg->time - crv_coinc.reco->time < 450) { return true; }
+bool track_crv_coincidence(TrackSegment& segment, CrvCoinc& crv_coinc) { 
+  if ( abs(segment.trkseg->time - crv_coinc.reco->time) > 150 ) { return true; }
```

Comment:
Is the sign the right way around here? This is saying if the time difference is more than 150 ns, then it's true that there's a tracker-CRV coincidence. Maybe there's another way of reading the function name so we might want to change it

Reply:
yes, I think its just a difference in how I was using it. I will reverse the sign for it to make more sense